wiki/models/article.py

```diff
@@ -169,7 +169,6 @@ def __unicode__(self):
         return _(u'Article without content (%(id)d)') % {'id': self.id}
 
     class Meta:
-        app_label = settings.APP_LABEL
```

Comment:
Why are all of these being removed? Does this lead to more of the warnings we all love to see when starting up devstack?

`RemovedInDjango19Warning: Model class <whatever> doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.`